x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_utils/validate_job_selection.ts

```diff
@@ -34,7 +34,27 @@ export function validateJobSelection(
   // (e.g. if switching to this view straight from the Anomaly Explorer).
   const invalidIds: string[] = difference(selectedJobIds, timeSeriesJobIds);
   const validSelectedJobIds = without(selectedJobIds, ...invalidIds);
-  if (invalidIds.length > 0) {
+
+  // show specific reason why we can't show the single metric viewer
+  if (invalidIds.length === 1) {
```

Comment:
It would be good to extend this functionality to the Anomaly Explorer to explain why we aren't showing anomaly charts for certain detectors.

Reply:
Added here https://github.com/elastic/kibana/pull/84594/commits/65ac8a9c1a45a878fedc9b562774107638bc760d

This will try to combine the message by error type (e.g. in this case, both records are linked to jobs with nested terms aggregations with model plot disabled)
<img width="1084" alt="Screen Shot 2020-12-06 at 19 33 07" src="https://user-images.githubusercontent.com/43350163/101308548-b65ac900-380f-11eb-8b5c-e2f4ff81c41f.png">

Another alternative is to show a call out for each job that we cannot show:
<img width="1775" alt="Screen Shot 2020-12-06 at 18 31 15" src="https://user-images.githubusercontent.com/43350163/101308594-d9857880-380f-11eb-825b-2c04b0b70b9f.png">

I prefer the first option, what do you think?

Reply:
I prefer the first option too, combining the error messages.